src/NServiceBus.Core/StaticHeaders/StaticHeadersConfigExtensions.cs

```diff
@@ -0,0 +1,33 @@
+namespace NServiceBus
+{
+    using NServiceBus.StaticHeaders;
+
+    /// <summary>
+    /// Extensions to the public configuration api
+    /// </summary>
+    public static class StaticHeadersConfigExtensions
+    {
+        /// <summary>
+        /// Allows static outgoing headers to be set
+        /// </summary>
+        /// <param name="config">Configuration object</param>
+        /// <param name="key">The header key</param>
+        /// <param name="value">The header value</param>
+        public static void SetOutgoingHeader(this BusConfiguration config, string key, string value)
```

Comment:
Maybe a better name ?
A lot of users get confused with the usage of these headers, should we name them something like `SetEndpointWideHeader` or something similar? Also api doco needs a bit more love IMO ;)